components/org.wso2.carbon.deployment.notifier/src/test/java/org/wso2/carbon/deployment/notifier/BaseTest.java

```diff
@@ -31,6 +35,8 @@
      * Basedir for all file I/O.
      */
     public static String basedir;
+    private static final String OS_NAME_KEY = "os.name";
+    private static final String WINDOWS_PARAM = "indow";
```

Comment:
There can be case mismatch when we check for "windows", if os.name is "Windows ...". So I skipped the first letter.